class2.html

```diff
@@ -432,15 +440,20 @@ <h6 class="grey">The Homework was:</h6>
                     <p>For example:</p>
                     <pre><code  class="ruby">
                       if x > 10
-                      puts "x is greater than 10"
-                      if x < 20
-                      puts "but x is less than 20"
-                      end
-                      elsif x <= 10 && x > 0
-                      puts "x is a number between 1 and 10"
+                        puts "x is greater than 10"
+                      elsif x < 20
+                        puts "but x is less than 20"
+                      elsif 0 < x && x <= 10
+                        puts "x is a number between 1 and 10"
                       else
-                      puts "Wow, don't be so negative, dude"
+                        puts "Wow, don't be so negative, dude"
                       end
+
+                      # Note!
+                      # You must use separate expressions:
+                      # You might be tempted, instead of  0 < x && x <= 10
+                      # to write 0 < x <= 10 (like you might in math)
+                      # This will cause errors.
```

Comment:
I don't think it's necessary to dwell too long on it (mostly because of the amount of info we have to cover). Since it's a pretty common mistake, I figure it's worth mentioning it since someone's bound to do it on their own, and it's in writing so if they refer back to the slides they'll see it.